settings.py

```diff
@@ -0,0 +1,142 @@
+import os.path
+import dj_database_url
+
+
+PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ['SECRET_KEY']
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = False
+
+TEMPLATE_DEBUG = False
+
+ALLOWED_HOSTS = ['seads.herokuapp.com']
+
+
+# Application definition
+TEMPLATE_CONTEXT_PROCESSORS = (
+    'django.contrib.auth.context_processors.auth',
+    'django.core.context_processors.request',
+)
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'rest_framework',
+    'rest_framework_swagger',
+    'seadssite',
+    'SeadsFront',
+    'api',
+)
+
+MIDDLEWARE_CLASSES = (
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+)
+
+ROOT_URLCONF = 'SeadsFront.urls'
+
+WSGI_APPLICATION = 'SeadsFront.wsgi.application'
+
+USERNAME = os.environ['USERNAME']
+PASSWORD = os.environ['PASSWORD']
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'mydb',
+        'USER': USERNAME,
+        'PASSWORD': PASSWORD,
+        'HOST': 'localhost',
+        'PORT': '',
+    }
+}
+
+# Update database configuration with $DATABASE_URL.
+
+db_from_env = dj_database_url.config(conn_max_age=500)
+DATABASES['default'].update(db_from_env)
+
+
+MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles','images')
+MEDIA_URL = '/images/'
+
+LANGUAGE_CODE = 'en-us'
+
+TIME_ZONE = 'UTC'
+
+USE_I18N = True
+
+USE_L10N = True
+
+USE_TZ = True
+
+
+STATIC_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles')
+STATIC_URL = '/staticfiles/'
+
+STATICFILES_DIRS = (
+    os.path.join(PROJECT_ROOT, 'seadssite', 'static'),
+    os.path.join(PROJECT_ROOT, 'SeadsFront', 'static'),
+
+)
+
+TEMPLATE_DIRS = (
+    os.path.join(
+        os.path.dirname(__file__),
+        'seadssite/templates/',
+    ),
+    os.path.join(
+        os.path.dirname(__file__),
+        'SeadsFront/templates/',
+    ),
+)
+
+# Simplified static file serving.
+# https://warehouse.python.org/project/whitenoise/
+
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+
+
+LOGIN_REDIRECT_URL = '/'
+
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'seadssystems@gmail.com'
```

Comment:
Better not to put emails in the code. Maybe make this an environmental variable as well?